client/tests/AccountForm.test.js

```diff
@@ -0,0 +1,174 @@
+import {h} from 'preact';
+import {shallow} from 'preact-render-spy';
+import AccountForm from '../src/components/AccountForm';
+import {REGISTER_PATH, LOGIN_PATH, RESET_PATH} from '../config';
```

Comment:
just noticed this wouldn't this change here with the base_endpoints refactor as well